### ISSUE TITLE
Creation failure - The "file" argument must be of type string

### ISSUE THREAD

**Issue by James-Quigley** [palmerhq/tsdx#55]
### Current Behavior

When running `npx tsdx create myapp`, the installation fails with the error:
```
✔ Created myapp
✖ Failed to install dependencies
TypeError [ERR_INVALID_ARG_TYPE]: The "file" argument must be of type string. Received type object

    at validateString (internal/validators.js:125:11)
    at normalizeSpawnArguments (child_process.js:399:3)
    at Object.spawn (child_process.js:532:38)
    at Object.module.exports [as default] (/Users/james.quigley/.npm/_npx/49155/lib/node_modules/tsdx/node_modules/execa/index.js:205:26)
    at prog.version.command.describe.action (/Users/james.quigley/.npm/_npx/49155/lib/node_modules/tsdx/dist/index.js:172:30)
```

### Expected behavior

Should successfully generate the full template, install deps, etc.

### Suggested solution(s)

Seems like a type error. Can investigate more later unless someone beats me to it

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| TSDX           | 0.5.0
| TypeScript       |
| Browser          |
| npm/Yarn         | 6.9/1.15.2
| Operating System | MacOS Mojave

**Comment by jaredpalmer** [comment]
Update 0.5.1

**Comment by jaredpalmer** [comment]
I will deprecate 0.5.0 on npm right now

**Comment by jaredpalmer** [comment]
Fixed in 30213c5
